src/twisted/internet/defer.py

```diff
@@ -1809,8 +1822,8 @@ class _InternalInlineCallbacksCancelledError(Exception):
 # type note: "..." is used here because we don't have a better way to express
 #     that the same arguments are accepted by the returned callable.
 def inlineCallbacks(
-    f: Callable[..., Generator[Deferred[object], object, None]]
-) -> Callable[..., Deferred[object]]:
+    f: Callable[..., Generator[Any, Any, None]]
+) -> Callable[..., Deferred[Any]]:
```

Comment:
`object` ➜ `Any` because otherwise Mypy will complain about decorated functions take return a generator with a `None` send-type, or return a `Deferred[None]`, for example.

Reply:
Hrm. I must have undone whatever I did that was triggering this… I'll revert it.